experiments/golden-results/StratoX-summary.txt

```diff
@@ -43,56 +43,51 @@ Calling function: Do_Withed_Unit_Spec
 Error message: This type of library_unit is not yet handled
 Nkind: N_Generic_Package_Declaration
 --
-Occurs: 30 times
-Calling function: Process_Declaration
-Error message: Package declaration
-Nkind: N_Package_Declaration
---
 Occurs: 23 times
 Calling function: Process_Pragma_Declaration
 Error message: Unsupported pragma: Check
 Nkind: N_Pragma
 --
-Occurs: 20 times
-Calling function: Process_Declaration
-Error message: Exception declaration
-Nkind: N_Exception_Declaration
---
-Occurs: 20 times
+Occurs: 22 times
 Calling function: Process_Declaration
 Error message: Package body declaration
 Nkind: N_Package_Body
 --
-Occurs: 20 times
-Calling function: Process_Pragma_Declaration
-Error message: Unsupported pragma: Precondition
-Nkind: N_Pragma
---
-Occurs: 15 times
+Occurs: 21 times
 Calling function: Process_Declaration
 Error message: Unknown declaration kind
 Nkind: N_Null_Statement
 --
-Occurs: 15 times
-Calling function: Process_Statement
-Error message: Unknown expression kind
-Nkind: N_Object_Declaration
+Occurs: 20 times
+Calling function: Process_Declaration
+Error message: Exception declaration
+Nkind: N_Exception_Declaration
 --
-Occurs: 14 times
+Occurs: 19 times
 Calling function: Process_Declaration
 Error message: Generic instantiation declaration
 Nkind: N_Function_Instantiation
 --
-Occurs: 11 times
+Occurs: 15 times
+Calling function: Process_Statement
+Error message: Unknown expression kind
+Nkind: N_Object_Declaration
```

Comment:
I recall seeing a patch that resolved an error that looked like this on UKNI.  I'm not sure who wrote it; I think it was either you or @hannes-steffenhagen-diffblue .  If that has been merged, should we be seeing these?

Reply:
This doesn't look like this has been *introduced* by this PR, it's just moved in the file because of other changes in the missing feature list.